.golangci.yml

```diff
@@ -53,7 +53,7 @@ linters:
     # - godox # TODO: Do we want this?
     # - goerr113 # TODO: Fix and enable
     - gofmt
-    # - gofumpt # TODO: Do we want this?
+    - gofumpt # TODO: Do we want this?
```

Comment:
This is billed as being a bit stricter than gofmt. Based on the changes in the source, it doesn't seem that compelling to me.